internal/math/sort.go

```diff
@@ -0,0 +1,17 @@
+package math
+
+import "golang.org/x/exp/constraints"
+
+func Min[T constraints.Ordered](a, b T) T {
+	if a < b {
+		return a
+	}
+	return b
+}
+
+func Max[T constraints.Ordered](a, b T) T {
+	if a < b {
+		return b
+	}
+	return a
+}
```

Comment:
I needed max function, to my surprise, it is not present in the Go stdlib :-D

Reply:
Well it would be: `int(math.Min(float64(w.pollMax), float64(runtime.NumCPU()+1)))` which looks awful to be honest, tiny little bit of mathematic code can't hurt any project is what I say :-)

Reply:
Can't hurt unless there is a bug in it :stuck_out_tongue: every additional line increases the chance of a bug is what I'd say :smile:
But in this case I agree yours looks bit tidier, so I'd be fine with having it, unless @avitova insists :shrug: